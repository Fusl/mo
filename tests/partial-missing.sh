#!/usr/bin/env bash

cd "${0%/*}"
../mo partial-missing.template 2>&1

if [[ $? -ne 1 ]]; then
    echo "Did not return 1"
fi
