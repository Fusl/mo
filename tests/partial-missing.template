Won't be there: {{> partial-missing.partial}}
