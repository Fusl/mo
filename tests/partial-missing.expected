cat: partial-missing.partial: No such file or directory
